plugins/inputs/sqlserver/sqlserverqueries.go

```diff
@@ -331,13 +342,19 @@ SELECT DISTINCT
 			,'Readahead pages/sec'
 			,'Lazy writes/sec'
 			,'Checkpoint pages/sec'
+			,'Free pages'
+			,'Extension free pages'
```

Comment:
Is this really a need? Aka is only for BP extensions which aren't used much

Reply:
This was based on a request. However, since these changes have been pulled out of this PR, I will address this offline or in the next PR.